src/coreclr/jit/valuenum.h

```diff
@@ -714,6 +721,18 @@ class ValueNumStore
     // Returns true iff the VN represents a (non-handle) constant.
     bool IsVNConstant(ValueNum vn);
 
+    // Returns true if the VN represents a Vector64/128/256.Zero.
+    bool IsVNVectorZero(ValueNum vn);
+
+#if FEATURE_SIMD
+    /// Returns VNSimdTypeInfo(0, CORINFO_TYPE_UNDEF) if the given value number has not been given a SIMD type.
+    VNSimdTypeInfo GetSimdTypeOfVN(ValueNum vn);
+
+    /// Returns VNSimdTypeInfo(0, CORINFO_TYPE_UNDEF) if the given value number has not been given a SIMD type
+    /// for a Vector.Zero value number.
+    VNSimdTypeInfo GetVectorZeroSimdTypeOfVN(ValueNum vn);
+#endif
```

Comment:
If you'll be adding full function headers, don't forget to remove the comments here.

(Note: we're moving away from these non-standard "small headers" in VN)